### PR TITLE
Update pl.json

### PR DESCRIPTION
**Pull request by LocalHeroPro**

### CODE REVIEW

File: locales/pl/pl.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "All rights reserved.": "Wszelkie prawa zastrzeżone",
     "Already registered?": "Zarejestrowany już?",
     "American Samoa": "Samoa Amerykańskie",
-    "An error occured while uploading the file.": "Wystąpił błąd podczas przesyłania pliku.",
+    "An error occurred while uploading the file.": "Wystąpił błąd podczas przesyłania pliku.",
```
Comment (caouecs) [inline locales/pl/pl.json:38]: Don't update tags please

**Review by caouecs — CHANGES_REQUESTED** [review]

File: locales/pl/pl.json (hunk 1 above)
Comment (LocalHeroPro) [reply locales/pl/pl.json:38]: I'm only asking: it's correct with one `r`, or this is typo?
Google Translate see it as typo:
image
Only asking ;]

Comment (caouecs) [reply locales/pl/pl.json:38]: It's not correct but the original file has the error. It's just a tag, so it's not important

Comment (LocalHeroPro) [reply locales/pl/pl.json:38]: So from where that original file comes from? I'll report that to author.

Comment (luisprmat) [reply locales/pl/pl.json]: This PR must be sent to Laravel, if they correct it there, we correct us here!

**Comment by caouecs** [comment]
Thank you

**Pull request merged**